pages/intermediate-web-lit.jsx

```diff
@@ -0,0 +1,136 @@
+var React = require('react');
+var Link = require('react-router').Link;
+var OutboundLink = require('react-ga').OutboundLink;
+
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+
+var Intro = (
+  <div>
+    <h1>Intermediate Web Literacy 1: Intro to CSS</h1>
+    <section className="intro">
+      <Illustration
+      height={204} width={204}
+      src1x="/img/pages/intermediate-web-lit/blocks.png"
+      alt="Multi-colored stacked blocks">
+        <h2>Learn to style HTML elements on a webpage using CSS selectors, attributes, and values.</h2>
+      </Illustration>
+    </section>
+  </div>
+);
+
+var IntermediateWebLit = (
+  <section className="row web-lit-basics">
+    <div className="col-sm-12">
+      <h2>Learning Objectives</h2>
+      <p>
+        Learners will understand and apply coding/scripting, composing, and design with CSS.
+      </p>
+    </div>
+  </section>
+);
+
+var curriculumList = [
+  {
+    title: "Writing the Web",
```

Comment:
just double checking since it's mentioned on the doc - do all modules listed here belong to "Writing the Web"?


<img width="835" alt="screen shot 2016-03-08 at 6 52 52 pm" src="https://cloud.githubusercontent.com/assets/2896608/13623955/38c639be-e55f-11e5-8c02-b339bfd73bd6.png">